README.md

```diff
@@ -118,6 +124,16 @@ version installed, you can replace `./gradlew` with `gradle` in the following st
    2d. Compile and run signal-cli:
 
         ./gradlew run --args="--help"
+3. Now you can copy the distribution that you just build, with:
+```   
+        cd build/distributions
+        # Unpack the compressed .tar file to the /opt directory.
+        sudo tar xf signal-cli-0.*.*.tar -C /opt
+        # Move the unpacked distribution into the /usr/local/bin/ directory
+        sudo ln -sf /opt/signal-cli-0.*.*/bin/signal-cli /usr/local/bin/
+```  
+4. You can verify `signal-cli` works, by typing: `signal-cli -v` which should return something like: `signal-cli 0.x.y`.
+5. Now you can choose to keep your phone registered and add this device using [these instructions](https://github.com/AsamK/signal-cli/wiki/Linking-other-devices-%28Provisioning%29). Or you can unregisters Signal on your phone, if you used Signal on your phone, and use this device as your main Signal device. The latter can be done either using the usage instructions above (`signal-cli -a ACCOUNT register` etc.), or, if you are prompted to fill in a captcha, using [these instructions](https://github.com/AsamK/signal-cli/wiki/Registration-with-captcha).
```

Comment:
Point 5 would fit better in the Usage section above. It's not related to building signal-cli.